From: Benjamin Drung <bdrung@ubuntu.com>
Date: Thu, 9 Jun 2022 15:32:50 +0200
Subject: Default to Ubuntu crash DB

---
 etc/apport/crashdb.conf | 30 +++++++++++++++++-------------
 1 file changed, 17 insertions(+), 13 deletions(-)

diff --git a/etc/apport/crashdb.conf b/etc/apport/crashdb.conf
index 0e7a71a..37fc9b8 100644
--- a/etc/apport/crashdb.conf
+++ b/etc/apport/crashdb.conf
@@ -1,6 +1,18 @@
 # map crash database names to CrashDatabase implementations and URLs
 
-default = 'debug'
+default = 'ubuntu'
+
+def get_oem_project():
+    '''Determine OEM project name from Distribution Channel Descriptor
+
+    Return None if it cannot be determined or does not exist.
+    '''
+    try:
+        dcd = open('/var/lib/ubuntu_dist_channel').read()
+        if dcd.startswith('canonical-oem-'):
+            return dcd.split('-')[2]
+    except IOError:
+        return None
 
 databases = {
     'ubuntu': {
@@ -11,18 +23,10 @@ databases = {
         'escalation_tag': 'bugpattern-needed',
         'escalated_tag': 'bugpattern-written',
     },
-    'fedora': {
-         # NOTE this will change Fall '07 when RHT switches to bugzilla 3.x!
-        'impl': 'rhbugzilla',
-        'bug_pattern_url': 'http://qa.fedoraproject.org/apport/bugpatterns.xml',
-        'distro': 'fedora'
-    },
-    'debian': {
-        'impl': 'debian',
-        'distro': 'debian',
-        'smtphost': 'reportbug.debian.org',
-        'recipient': 'submit@bugs.debian.org',
-        'sender': ''
+    'canonical-oem': {
+        'impl': 'launchpad',
+        'bug_pattern_url': 'http://people.canonical.com/~ubuntu-archive/bugpatterns/bugpatterns.xml',
+        'project': get_oem_project(),
     },
     'debug': {
         # for debugging
